### ISSUE TITLE
cannot launch ytmdesktop anymore

### ISSUE THREAD

**Issue by akabakass** [adlerluiz/ytmdesktop#1432]
### Consent

- [X] I verified that there is no open/closed issue for the same subject.
- [X] I understand that **YTMDesktop have NO affiliation with Google or YouTube**

### Current Behavior

youtube desktop is not launching

### Expected Behavior

youtube desktop starts

### Steps To Reproduce

I don't really have steps to reproduce, I think it's related to when I've updated to ubuntu 24.04.
when I launch from cli, I only get this:
```
[32663:0904/100942.035573:FATAL:credentials.cc(127)] Check failed: . : Permission non accordée (13)
Trappe pour point d'arrêt et de trace (core dumped)
```
I'm on V2.0.5, I've tried downgraded to V2.0.0 with no luck.
I'm installing with the released DEB version.
using the 1.14 old version snap works for me for now, with all the problems I used to have of course.

### YTMDesktop

V2.0.X

### OS

ubuntu

### OS Version

24.04

### Arch

x64

### Installation way

deb

**Comment by Alipoodle** [comment]
This appears that permission to something like your Credencials Manager was denied?

Have you added any additional paramaters to change and settings for the Safe Storage?
https://www.electronjs.org/docs/latest/api/safe-storage#safestoragegetselectedstoragebackend-linux

**Comment by Alipoodle** [comment]
https://stackoverflow.com/questions/76165512/fatalcredentials-cc127-check-failed-permission-denied-13-trace-breakpo seems to be a similar case to you?

**Comment by akabakass** [comment]
Thanks for the quick response, I'm not really using electron, node or anything like that.
And I don't think I've ever played with that.
I've tried running the command you pointed from REPL.
first as my user using `.bin/electron --interactive` gave me the same error
trying to run that as sudo asked me to use --no-sandbox, fair enough.
and using that allows me to run `safeStorage.getSelectedStorageBackend()` which return 'basic_text'
after digging a little bit, I've found a workaround here
https://portal.perforce.com/s/article/Resolve-an-apparmor-DENIED-error-message-in-P4V
using
`sudo sysctl -w kernel.apparmor_restrict_unprivileged_userns=0`
